src/js/components/DataTable/Resizer.js

```diff
@@ -51,48 +59,62 @@ const Resizer = ({ onResize, property, headerText, messages, headerId }) => {
     }
   }, [ref]);
 
-  const onMouseDown = useCallback((event) => {
+  useEffect(() => {
```

Comment:
there is a `useEffect` above that sets the initial width, can we not combine the 2 `useEffects` or do we need both?

Reply:
thank you!